core/trino-main/src/main/java/io/trino/sql/planner/ConnectorExpressionTranslator.java

```diff
@@ -43,6 +43,8 @@
 import io.trino.sql.tree.DoubleLiteral;
 import io.trino.sql.tree.Expression;
 import io.trino.sql.tree.FunctionCall;
+import io.trino.sql.tree.InListExpression;
+import io.trino.sql.tree.InPredicate;
```

Comment:
add tests in `TestConnectorExpressionTranslator`

i recommend to use `assertTranslationRoundTrips` being added in #11086  to keep tests shorter.

Reply:
please split change into two commits, one for engine side, and the other for PostgreSQL side